OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -42,6 +42,7 @@
     <ProjectReference Include="..\OpenRA.Game\OpenRA.Game.csproj">
       <Private>False</Private>
     </ProjectReference>
+    <PackageReference Include="DiscordRichPresence" Version="1.0.150" />
```

Comment:
This should now be
```xml
    <Reference Include="DiscordRichPresence">
      <HintPath>..\thirdparty\download\DiscordRPC.dll</HintPath>
      <Private>False</Private>
    </Reference>
```
and ideally included in the `ItemGroup` above with the other references.